posts/Yoann/2019-03-22-hibernate-search-6-0-0-Alpha3.adoc

```diff
@@ -0,0 +1,253 @@
+between the Elasticsearch and Lucene backend: https://hibernate.atlassian.net/browse/HSEARCH-3482[HSEARCH-3482].
+
+=== Documentation
+
+The documentation is still sparse, but it continues to grow.
+We documented in particular the configuration properties
+(https://hibernate.atlassian.net/browse/HSEARCH-3473[HSEARCH-3473],
+see https://docs.jboss.org/hibernate/search/6.0/reference/en-US/html_single/#configuration[here])
+and the limitations of using the data/time types from `java.util`
+(https://hibernate.atlassian.net/browse/HSEARCH-3509[HSEARCH-3509],
+see https://docs.jboss.org/hibernate/search/6.0/reference/en-US/html_single/#mapper-orm-legacy-date-time-apis[here]).
+
+=== Version upgrades
+
+* https://hibernate.atlassian.net/browse/HSEARCH-3483[HSEARCH-3483]:
+Upgrade to Lucene 7.7.0
+* https://hibernate.atlassian.net/browse/HSEARCH-3520[HSEARCH-3520]:
+Upgrade to Elasticseearch 6.6.2
+* https://hibernate.atlassian.net/browse/HSEARCH-3490[HSEARCH-3490]:
+There is now experimental support for Elasticsearch 7.0.0-beta1.
+* https://hibernate.atlassian.net/browse/HSEARCH-3514[HSEARCH-3514]:
+We restored automatic Java module names to our JARs, which can thus be safely referenced from your `module-info.java`.
+* https://hibernate.atlassian.net/browse/HSEARCH-3485[HSEARCH-3485],
+https://hibernate.atlassian.net/browse/HSEARCH-3493[HSEARCH-3493]:
+Hibernate Search still targets JDK8 and JDK11, but now we also test compatibility with JDK12 and JDK13 regularly.
+
+=== Other improvements and bug fixes
+
+* https://hibernate.atlassian.net/browse/HSEARCH-3257[HSEARCH-3257]: Allow to bypass bridges in projections
+* https://hibernate.atlassian.net/browse/HSEARCH-2658[HSEARCH-2658]: Support @Inject in all user-provided components: bridges, configurers, ...
+* https://hibernate.atlassian.net/browse/HSEARCH-1640[HSEARCH-1640]: boolean field should never be analyzed
+* https://hibernate.atlassian.net/browse/HSEARCH-2908[HSEARCH-2908]: Incorrect mapping for 'java.lang.Short and java.lang.Byte' fields with Elasticsearch.
+* https://hibernate.atlassian.net/browse/HSEARCH-3481[HSEARCH-3481]: Change max_connections_per_route default value to 10
+* https://hibernate.atlassian.net/browse/HSEARCH-3251[HSEARCH-3251]: Avoid "fuzzy" query in Elasticsearch and prefer "match" query with "fuzziness" parameter
+* https://hibernate.atlassian.net/browse/HSEARCH-2248[HSEARCH-2248]: Remove deprecated threshold method for fuzzy queries from the DSL
+* https://hibernate.atlassian.net/browse/HSEARCH-3477[HSEARCH-3477]: Wrong incompatible types error when query a full text field across different indexes on Lucene backend
+* https://hibernate.atlassian.net/browse/HSEARCH-3489[HSEARCH-3489]: Improve the error message when the inverse side of an association cannot be found by Hibernate Search
+* https://hibernate.atlassian.net/browse/HSEARCH-3529[HSEARCH-3529]: MassIndexer fails for entities with primitive ID type
```

Comment:
I would mention that we have now a nice Spring Boot quickstart to show, the showcase.